### PR TITLE
Clients list fragment

### LINKED ISSUE #16: Clients list Fragment
- search input
- add button

### PR DESCRIPTION
**Pull request by wiktoriasalamon**
Closes #16

### CODE REVIEW

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/MainActivity.kt [hunk 1]
```diff
@@ -19,4 +27,4 @@ class MainActivity : AppCompatActivity() {
         super.onDestroy()
         finish()
     }
-}
\ No newline at end of file
+}
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/MainActivity.kt]:
```suggestion
}

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_clients, container, false)
+        clientsListView = view.findViewById(R.id.clients_list_view) as ListView
+        clientsList = listOf("Jan Kowalski", "Anna Nowak", "Katarzyna Nosowska")
+        clientsListAdapter = ClientsListAdapter(this.context!!, clientsList)
+        clientsListView.adapter = clientsListAdapter
+
+        return view
+    }
+}
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]:
```suggestion
}

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsListAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            view.tag=viewHolder
+        }
+
+        val holder = view.getTag() as ViewHolder
+
+        holder.nameTextView!!.text=data[position]
+
+        return view
+    }
+}
```
Comment (wiktoriasalamon) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsListAdapter.kt]:
```suggestion
}

```

File: app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+class ClientsFragment : Fragment() {
+    lateinit var clientsListAdapter: ClientsListAdapter
+    lateinit var clientsListView : ListView
+    lateinit var clientsList: List<String>
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_clients, container, false)
+        clientsListView = view.findViewById(R.id.clients_list_view) as ListView
+        clientsList = listOf("Jan Kowalski", "Anna Nowak", "Katarzyna Nosowska")
```
Comment (maclewan) [inline app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]: Move gaining clients list to separate procedure!!!!!111oneone

Comment (wiktoriasalamon) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]: done

Comment (wiktoriasalamon) [reply app/src/main/java/com/piwniczna/mojakancelaria/activities/clients/ClientsFragment.kt]: @sasuke5055

**Review by maclewan — COMMENTED** [review]
ok

**Pull request merged**